app/jobs/candidates/registrations/placement_request_job.rb

```diff
@@ -0,0 +1,14 @@
+module Candidates
+  module Registrations
+    class PlacementRequestJob < ApplicationJob
+      queue_as :default
+
+      retry_on \
+        Notify::RetryableError, wait: :exponentially_longer, attempts: 5
```

Comment:
This will need to be much longer - by my maths this will retry 4**4 + 2 = 258 seconds, so approx 4 minutes which doesn't all for much downtime.

one option is a custom proc for the wait delay and something more aggressive - maybe something like `attempts ** 8` ?

Another is simply to set a much higher retry attempts (eg 12 attempts)

11**4 + 2 = approx 4 hours? 

At that point we'll get notified that the job failed